actions/kpops-runner/action.yaml

```diff
@@ -25,16 +25,30 @@ inputs:
     required: false
   kpops-version:
     description: "KPOps version"
-    required: true
+    required: false
+    default: "latest"
+  helm-version:
+    description: "Helm version"
+    required: false
+    default: "latest"
+  token:
+    description: "secrets.GITHUB_TOKEN, needed for setup-helm action if helm-version is set to latest"
+    required: false
 
 runs:
   using: "composite"
   steps:
+    - name: Set up Helm
+      uses: azure/setup-helm@v3
+      with:
+        version: ${{ inputs.helm-version }}
+        token: ${{ inputs.token }}
```

Comment:
Do you know why a token is needed? Seems quite unintuitive

Reply:
idk, it's written in their readme
actually I had a look in their code just now and it seems the actual latest version is hardcoded https://github.com/Azure/setup-helm/blob/ac5ee1fca8fba06d82bfc39af726f19ffe037a49/src/run.ts#L16